.github/stale.yml

```diff
@@ -1,8 +1,8 @@
 # Probot Stale configuration file
 
 # Number of days of inactivity before an issue becomes stale
-# 1030 is approximately 2 years and 10 months
-daysUntilStale: 1030
+# 1000 is approximately 2 years and 9 months
+daysUntilStale: 1000
```

Comment:
Any specific reason this is made 2 years ? That seems awfully long for any bug to be fixed ?

Reply:
We have a large backlog of issues, as you know, so this is a work in progress. We decided to gradually reduce the `daysUntilStale` value over time. See [this list of PRs](https://github.com/saltstack/salt/pulls?q=is%3Apr+stale+is%3Aclosed+author%3Arallytime) as an example.

Once we get to a more reasonable number, the value will remain static.

Reply:
👍